MVVMRxSwiftDemo/Sources/Presentation/Views/AnswerView/RatingAnswerView.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  RatingAnswerView.swift
+//  MVVMRxSwiftDemo
+//
+//  Created by Chananchida F. on 7/29/21.
+//
+
+import UIKit
+import RxSwift
+import RxGesture
+
+class RatingAnswerView: AnswerView {
+    
+    private var icon: String
+
+    private let stackView = UIStackView()
+    
+    private let disposeBag = DisposeBag()
+
+    init(icon: String) {
+        self.icon = icon
+        super.init(frame: .zero)
+        setUpLayout()
+        setUpViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func configure(with answers: [SurveyAnswer]) {
+        stackView.arrangedSubviews.forEach {
+            if let iconLabel = $0 as? UILabel {
+                iconLabel.text = icon
+            }
```

Comment:
let's use guard let for left side happy flow
```suggestion
            guard let iconLabel = $0 as? UILabel else { return }
            iconLabel.text = icon
```